Makefile

```diff
@@ -1,58 +1,65 @@
+.PHONY: \
+	force-rebuild \
+	install \
+	tests-locally \
+	lint-locally \
+	clean \
+	images \
+	tests \
+	lint \
+	lint-errors \
+	tests8 \
+
 # Project constants
 IMAGE ?= convert2rhel
 PYTHON ?= python3
 PIP ?= pip3
 VENV ?= .venv3
 
-.PHONY: all
 all: clean images tests
 
-.PHONY: install
-install:
+install: .install
+
+.install:
```

Comment:
Could you explain what this means? Why would this only run once if you call tests? Due to `touch`?

Reply:
as far as I know, the target `install` requires as a dependency a not PHONY target (which is the file or another target). Once we touch the target name (touch $@) then a file is created and next call to make tests will find that the file dependency (.images) is presented. Otherwise it will run the images target

Reply:
more interesting example is here https://www.cmcrossroads.com/article/rebuilding-when-files-checksum-changes

Reply:
Resolved @SpyTec ?